ui/src/app/base/components/StatusBar/StatusBar.tsx

```diff
@@ -34,17 +37,48 @@ const getLastCommissionedString = (machine: MachineDetails) => {
   }
 };
 
+const getSyncStatusString = (syncStatus: string) => {
+  if (syncStatus === "") {
+    return "Never";
+  }
+  try {
+    return getTimeDistanceString(syncStatus);
+  } catch (error) {
+    return `Unable to parse sync timestamp (${
+      error instanceof Error ? error.message : error
+    })`;
+  }
+};
+
 export const StatusBar = (): JSX.Element | null => {
   const activeMachine = useSelector(machineSelectors.active);
   const version = useSelector(versionSelectors.get);
   const maasName = useSelector(configSelectors.maasName);
+  const isHardwareSyncEnabled =
+    activeMachine &&
+    "enable_hw_sync" in activeMachine &&
+    activeMachine.enable_hw_sync === true;
```

Comment:
We typically use the `isMachineDetails()` util now.

``` ts
const isHardwareSyncEnabled =
  isMachineDetails(activeMachine) &&
  activeMachine.enable_hw_sync === true;
```

Reply:
sure! Wille use this instead. Thanks!